inference-engine/src/cldnn_engine/ops/parameter.cpp

```diff
@@ -174,107 +177,129 @@ static void CreateParameterOp(Program& p, const std::shared_ptr<ngraph::op::v0::
         break;
     }
 
-    if (ColorFormat::NV12 == preProcess.getColorFormat() && p.GetConfig().nv12_two_inputs) {
-        // for NV12, create two input layouts with reorder instead of one,
-        // and then would expect compound blob in inferRequest
-        if (Layout::NCHW != l &&
-            (Precision::I8 != ip || Precision::U8 != ip)) {
-            IE_THROW() << "Unsupported layout (" << l << ") or precision "
-                               << ip.name() << ") for NV12 input " + inputInfo->name();
+    bool is_convert_color_input = false;
+    for (auto& node : op->get_users()) {
+        std::string node_type = node->get_type_name();
+        is_convert_color_input |= node_type == "NV12toRGB" || node_type == "NV12toBGR";//another way?
```

Comment:
My point that it's OK.

Reply:
`ngraph::is_type<ngraph::op::v8::NV12toRGB>() || ...`